test/Intl/CollatorOptions.js

```diff
@@ -108,3 +108,4 @@ var tests = [
 ];
 
 testRunner.runTests(tests, { verbose: WScript.Arguments[0] != "summary" });
+// assert.areEqual(new Intl.Collator("de-DE-u-co-phonebk", {}).compare("\u00e4b", "ada"), 1, "Comparing with collation unicode extension phonebk.");
```

Comment:
Is this broken now?

Reply:
Nope, it was a vestige of me testing one-off asserts. I need to remove it.